src/components/CreateCoursePage/index.jsx

```diff
@@ -11,6 +11,7 @@ class CreateCoursePage extends React.Component {
   constructor(props) {
     super(props);
     this.handleCourseCreate = this.handleCourseCreate.bind(this);
+    if (props.courseInfo.error) this.props.loadCreateNewCoursePage();
```

Comment:
Nit: Could we rename this method?

`loadCreateNewCoursePage`

Maybe `initializeCreateCourse` or `clearErrors`

Reply:
I'm hesitant to just do clearErrors because it's specifically clearing courseInfo errors. Any thoughts between `initializeCreateCourse` / `initializeCreateCourseRun` or `clearCourseInfoErrors`? @staubina